src/components/tipRecords/TipRecord.vue

```diff
@@ -226,25 +233,28 @@ export default {
       };
       await Backend.claimFromUrl(postData);
     },
-    pinOrUnPinTip(pinTip = true) {
-      if (!this.account) {
-        return;
-      }
-      Backend.pinOrUnPinItem(
-        this.tip.id,
-        'TIP',
-        this.account,
-        (data) => client.signMessage(data),
-        pinTip,
-      ).then(() => {
-        Backend.getPinnedItems(this.account)
-          .then((updatedPinnedItems) => {
-            this.$store.commit('setPinnedItems', updatedPinnedItems);
-          })
-          .catch(console.error);
-      }).catch((error) => {
-        console.log(error);
-      });
+    async sendReport() {
+      const postData = {
+        tipId: this.tip.id,
+        author: this.account,
+      };
+      const { challenge } = await Backend.sendPostReport(postData);
+      const url = new URL(`${window.location.origin}/tip/${this.tip.id}`);
```

Comment:
isn't it the same as the current url?

Reply:
> isn't it the same as the current url?

For example you could pin a tip from user-profile. Then the success callback would redirect to tip specific page so only this specific tip methods are executed.